### PR TITLE
SF-1927 Allow all users to update their display name

### PR DESCRIPTION
**Pull request by nigel-wells**
- Removed the block for accessing the update display name dialog
- Return type fixes

- - -
This change is [Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1760)

### CODE REVIEW

File: src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts [hunk 1]
```diff
@@ -576,6 +566,17 @@ describe('AppComponent', () => {
       verify(mockedUserService.editDisplayName(false)).once();
     }));
 
+    it('non auth0 connection users can edit name', fakeAsync(() => {
```
Comment (Nateowami) [inline src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts:569]: The phrasing "non auth0 connection" is kind of a misnomer; these users are connected via Auth0. What about specifically calling out Paratext users for this test?

**Review by Nateowami — APPROVED** [review]
Looks good; one suggested change to wording.

**Review by nigel-wells — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1760)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @Nateowami)

___
*[`src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts` line 569 at r1](https://reviewable.io/reviews/sillsdev/web-xforge/1760#-NRAdE7Fj7Ek0wgZFBeI-r1-569:-NRAx3wx-BhZbde8MxXQ:b-4uo6so) ([raw file](https://github.com/sillsdev/web-xforge/blob/30d68a4568bda998b625e64478179017cfdfeeef/src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts#L569)):*
<details><summary><i>Previously, Nateowami (Nathaniel Paulus) wrote…</i></summary><blockquote>

The phrasing "non auth0 connection" is kind of a misnomer; these users are connected via Auth0. What about specifically calling out Paratext users for this test?
</blockquote></details>

I've removed the other 2 tests in favor of just one

**Review by Nateowami — APPROVED** [review]
> Reviewed 2 of 3 files at r1, 1 of 1 files at r2, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/sillsdev/web-xforge/1760)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @nigel-wells)

___
*[`src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts` line 569 at r1](https://reviewable.io/reviews/sillsdev/web-xforge/1760#-NRAdE7Fj7Ek0wgZFBeI-r1-569:-NRB-IbWAPwLcxcSnjjF:b-kn3kdh) ([raw file](https://github.com/sillsdev/web-xforge/blob/30d68a4568bda998b625e64478179017cfdfeeef/src/SIL.XForge.Scripture/ClientApp/src/app/app.component.spec.ts#L569)):*
<details><summary><i>Previously, nigel-wells (Nigel Wells) wrote…</i></summary><blockquote>

I've removed the other 2 tests in favor of just one
</blockquote></details>

Looks good. Thanks.

**Pull request merged**